jplag/src/main/java/de/jplag/clustering/algorithm/AgglomerativeClustering.java

```diff
@@ -1,11 +1,6 @@
 package de.jplag.clustering.algorithm;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.HashSet;
-import java.util.List;
-import java.util.PriorityQueue;
-import java.util.Set;
+import java.util.*;
```

Comment:
Is the star import intended?

Reply:
Is automatically done if too many utils are imported :)

Reply:
Ah, then we might need to adjust the spotless config. At least I am more in favor of explicit imports, as modern IDEs allow to hide the imports.

Reply:
done by manually configuring ij. editorconfig did not work for me. spotless is not able to manage this.